### PR TITLE
Convert setup.sh to Makefile.

### PR DESCRIPTION
**Pull request by vext01**
When we did the warmup experiment, I always wished that I'd made `build.sh` a `Makefile` to avoid this idiom:
```
if [ ! -f renaissance-gpl-${RENAISSANCE_V}.jar ]; then
    wget https://github.com/renaissance-benchmarks/renaissance/releases/download/v${RENAISSANCE_V}/renaissance-gpl-${RENAISSANCE_V}.jar
fi
```

Make's dependency tracking is better suited (and more concise) for downloading/untarring things if they don't exist.

Also add targets to clean and run the experiment too.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+GRAALCE_V=1.0.0-rc16
+GRAALCE_DIR=graalvm-ce-${GRAALCE_V}
+GRAALCE_TGZ=graalvm-ce-${GRAALCE_V}-linux-amd64.tar.gz
+
+all: setup
+
+setup: ${RENAISSANCE_JAR} ${GRAALCE_DIR}
+
+${RENAISSANCE_JAR}:
+	echo ${RENAISSANCE_JAR}
```
Comment (vext01) [inline Makefile]: Gah!

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+FETCH=wget
+
+RENAISSANCE_V=0.9.0
+RENAISSANCE_JAR=renaissance-gpl-${RENAISSANCE_V}.jar
+
+GRAALCE_V=1.0.0-rc16
+GRAALCE_DIR=graalvm-ce-${GRAALCE_V}
+GRAALCE_TGZ=graalvm-ce-${GRAALCE_V}-linux-amd64.tar.gz
+
+all: setup
```
Comment (ltratt) [inline Makefile]: This line feels redundant: if "all" this `Makefile` will ever do is build things, then we might as well have `setup` as the first target.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	echo ${RENAISSANCE_JAR}
+	${FETCH} https://github.com/renaissance-benchmarks/renaissance/releases/download/v${RENAISSANCE_V}/${RENAISSANCE_JAR}
+
+${GRAALCE_TGZ}:
+	${FETCH} https://github.com/oracle/graal/releases/download/vm-${GRAALCE_V}/${GRAALCE_TGZ}
+
+${GRAALCE_DIR}: ${GRAALCE_TGZ}
+	tar xfz ${GRAALCE_TGZ}
+
+run-standalone: setup
```
Comment (ltratt) [inline Makefile:30]: Can we use a shorter name here? I always find long `make` targets hard to remember :( Or maybe it's not even worth having this as a target...

File: Makefile (hunk 2 above)
Comment (vext01) [reply Makefile]: Good point. Just `all` is a convention. I don't feel super strongly.

Comment (ltratt) [reply Makefile]: I think `all` is useful if it calls multiple targets; in this instance it feels a bit convention-for-the-sake-of-it TBH.

File: Makefile (hunk 3 above)
Comment (vext01) [reply Makefile:30]: I want to imply that we are running outside krun (there are two run modes).

From my POV, I don't care too much about the length of the target name. I just type `make run<tab>` and it completes. Even bash does this.

File: Makefile (hunk 2 above)
Comment (vext01) [reply Makefile]: Fair enough. I'll kill it. If another target comes along that we want to run by default then we can bring it back.

File: Makefile (hunk 3 above)
Comment (ltratt) [reply Makefile:30]: I guess my question is also whether we want to tell people to use a `make` target or a shell script.

Comment (vext01) [reply Makefile:30]: I've opted to tell them to use the `Makefile` (also stopped `run_benchmarks.sh` calling back into `make`).

It seems simpler for a user to have one place to look. I find this *can* break down if the user frequently needs to change command line arguments to things, but that doesn't seem to be the case here. The args seem fixed.

Comment (ltratt) [reply Makefile:30]: OK.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**